test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/LoadTest.java

```diff
@@ -46,7 +46,7 @@
 	public static OptionalInt hcsSubmitMessage = OptionalInt.empty();
 	public static OptionalInt hcsSubmitMessageSizeVar = OptionalInt.empty();
 	/** initial balance of payer account used for paying for performance test transactions */
-	public static OptionalLong initialBalance = OptionalLong.of(900_000_000_000L);
+	public static OptionalLong initialBalance = OptionalLong.of(90_000_000_000_000L);
```

Comment:
Since this is already `public`, we could use it for other test (`CryptoCreateForSuiteRunner` as below) to avoid duplicate/redeclaration.

Reply:
Fixed. Thanks Quan !